qiita_plugins/qiita_client/qiita_client/exceptions.py

```diff
@@ -0,0 +1,23 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014--, The Qiita Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+
+
+class QiitaClientError(Exception):
```

Comment:
They're used in the Qiita client and I think there are 3 different errors that the developer using the QiitaClient can decide what to do (either propagate the error or try something else). For example, this allows me to code a different behavior in the `_heartbeat` function below.